vmdk_plugin/main.go

```diff
@@ -97,8 +98,8 @@ func main() {
 	var driver volume.Driver
 
 	// Define command line options
-	driverName := flag.String("driver", "vmdk", "Volume driver")
-	logLevel := flag.String("log_level", "debug", "Logging Level")
+	driverName := flag.String("driver", vmdkDriver, "Volume driver")
```

Comment:
I also do not think running 2 process is a good idea - but can we just register the driver with the same socket twice ? Or can we register it twice and listen on both sockets from one driver ? Restarting the driver service to switch between supporting `-d vmdk` and `-d vsphere` seems pointless, if that's the only choice then I think we (anbd customers) would be better off if we  make a breaking change and stick with `vsphere`